index.html

```diff
@@ -17,15 +18,16 @@
         </div>
         <nav class="menu">
             <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
+            <i class="fa fa-close" id="close"></i>
```

Comment:
- [ ] OPTIONAL
Nice job with the hamburger, however, I we can further improve upon the **user experience** by ensuring that the _close button_ is behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _close button_ on _mouse hover_ events.